de.metas.fresh/de.metas.fresh.base/src/main/sql/postgresql/ddl/functions/Docs_Sales_InOut_Root.sql

```diff
@@ -32,7 +33,20 @@ $$SELECT
 	END AS printname,
 	CASE WHEN io.DocStatus IN ('DR', 'IP') THEN 'de_CH' ELSE $2 END AS AD_Language,
 	isHidePackingMaterialInShipmentPrint,
-	mb.email
+	mb.email,
+	CASE
+		WHEN
+		EXISTS(
+			SELECT 0
+			FROM M_InOutLine iol1
+			INNER JOIN M_Product p ON iol1.M_Product_ID = p.M_Product_ID AND p.isActive = 'Y'
```

Comment:
why do you only join the product if it is active? what's the benenig of that?
unfortunately i already integrated the other repo, so I'm now removing it. please contact me asap, so we might add it once again